reaper/cogs/log_reading.py

```diff
@@ -119,7 +120,7 @@ async def on_message(self, message):
         problem = discord.Embed(
             title="Problems I found in your log:", description="", color=0x5D3FD3
         )
-        dm_log = discord.Embed(
+        tracker = discord.Embed(
```

Comment:
Probably wanna go with another name than `tracker` cause I can already envision someone seeing this and going "Reaper is tracking and storing all my logs!!!!1!!!11!!1", when really all it does is link to the message containing the log :melting_face: